iceoryx_posh/include/iceoryx_posh/popo/publisher.hpp

```diff
@@ -34,7 +34,7 @@ class Publisher_t
     /// @param[in] service Information on service , service, instance, event Id
     /// @param[in] runnableName optional name of the runnable the publisher belongs to
     Publisher_t(const capro::ServiceDescription& service,
-                const cxx::CString100& runnableName = cxx::CString100("")) noexcept;
+                const RunnableName_t& runnableName = RunnableName_t("")) noexcept;
```

Comment:
Doesn't have the `string` a default ctor which constructs an empty string? It is just minor but I would write `RunnableName_t()`.
Also below `Publisher` ctor

Reply:
yes